rest-api/README.md

```diff
@@ -0,0 +1,120 @@
+# Sample code for fetching session data with Kobiton REST API
+## Prerequisites
+  - Node 
+    >Please visit here to install https://nodejs.org/en/download/.
+
+  - Kobiton Username and API key
+    >Please visit our [blog](https://kobiton.com/blog/tutorial/parallel-testing-selenium-webdriver/) and navigate to `IV. Configure Test Scripts for Kobiton` for more details in getting Username and API key.
+
+## Kobiton REST API command
+  This repository includes samples of simple NodeJS app for fetching test session data with Kobiton REST API.
+
+  The sample contains scripts:
+  - [get-session-data.js](https://github.com/kobiton/samples/rest-api/get-session-data.js): Fetch a session data
+  - [get-session-commands.js](https://github.com/kobiton/samples/rest-api/get-session-commands.js): Fetch all commands of a session
+
+> Note: Kobiton also provides other languages to fetch Kobiton REST API, visit [our docs](https://api.kobiton.com/docs) for more information.
+
+## Setup
+- Clone this repository with
+    ```
+      git clone git@github.com:kobiton/samples.git
+    ```
+
+- Open `rest-api` folder.
+
+## Get sessions data
+
+- Request command: 
+
+    `GET /sessions/{sessionId}`
+
+ Retrieve a session info belonging to the current user or organization.
+> Note: Absolute urls in response body like *"screenshotDownloadUrl"*, *"previewUrl"*, etc. expire in 2 hours since the response is made.
+
+
+- Execute: 
+    ```bash
+    $ USERNAME=<YOUR_KOBITON_USERNAME> \
+    > API_KEY=<YOUR_KOBITON_API_KEY> \
+    > SESSION_ID=<SESSION_ID> \
+    > node get-session-data.js 
+    ```
+  **Example:**
```

Comment:
I just think that we don't need to bold this, what do you think ? 

```
**bold**
notbold
```
- Result: 
**bold**
notbold

Reply:
yeah, it looks better